src_types/common/utilityTypes.d.ts

```diff
@@ -0,0 +1,3 @@
+type IntersectionArrayAndBase<T, P> = {
```

Comment:
last time i'll say it... :)
no reason to have this file. $w is the only one using it and probably the only one that'll ever use it (or something else eventually). no reason for this extra file.

Reply:
Well, actually I already did it in `selector-declaration-builder.js` but I just forgot to remove this file :)